anago

```diff
@@ -620,9 +620,17 @@ prepare_tree () {
     logrun -s git commit -am "Delete extraneous CHANGELOG-*.md files on branch." || return 1
   fi
 
+  # This appears to be a bug fix for some historical bug.  Unclear if/why
+  # this would ever result in label != label_common.
   # Ensure a common name for label in case we're using the special beta indexes
   [[ "$label" =~ ^beta ]] && label_common="beta"
 
+  # Ensure any beta tag is on a unique commit by making a commit here:
+  if [[ "$label_common" == "beta" ]]; then
+    logecho "Making a unique commit for tag ${RELEASE_VERSION[$label]}"
+    logrun git commit --allow-empty -m "${RELEASE_VERSION[$label]}" || return 1
```

Comment:
```suggestion
    logrun git commit --allow-empty -m "Add tag for Kubernetes ${RELEASE_VERSION[$label]} (empty commit)" || return 1
```

Reply:
How verbose should we be?  Neither at the moment would say "why" and that is useful.

Perhaps "Work around k/release issue #1020 with tag ${RELEASE_VERSION[$label]}"